### PR TITLE
feat(CopyClipboard): apply tokens

### PR DESCRIPTION
**Pull request by andrew-ronaldson**
Closes https://github.com/patternfly/patternfly/issues/6194

### CODE REVIEW

File: src/patternfly/components/ClipboardCopy/clipboard-copy.scss [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
-  --#{$clipboard-copy}__expandable-content--PaddingTop: var(--#{$pf-global}--spacer--md);
-  --#{$clipboard-copy}__expandable-content--PaddingRight: var(--#{$pf-global}--spacer--md);
-  --#{$clipboard-copy}__expandable-content--PaddingBottom: var(--#{$pf-global}--spacer--md);
-  --#{$clipboard-copy}__expandable-content--PaddingLeft: var(--#{$pf-global}--spacer--md);
+  --#{$clipboard-copy}__expandable-content--MarginTop: var(--pf-t--global--spacer--xs);
+  --#{$clipboard-copy}__expandable-content--PaddingTop: var(--pf-t--global--spacer--md);
+  --#{$clipboard-copy}__expandable-content--PaddingRight: var(--pf-t--global--spacer--md);
+  --#{$clipboard-copy}__expandable-content--PaddingBottom: var(--pf-t--global--spacer--md);
+  --#{$clipboard-copy}__expandable-content--PaddingLeft: var(--pf-t--global--spacer--md);
   --#{$clipboard-copy}__expandable-content--BackgroundColor: var(--#{$pf-global}--BackgroundColor--light-100);
```
Comment (srambach) [inline src/patternfly/components/ClipboardCopy/clipboard-copy.scss:15]: I think this one should be a token; --pf-t--global--background--color--primary--default ?

**Review by srambach — COMMENTED** [review]
🥳 Looks great! Just one token missing I think.

File: src/patternfly/components/ClipboardCopy/clipboard-copy.scss [hunk 2]
```diff
@@ -59,6 +63,7 @@
 
 .#{$clipboard-copy}__group {
   display: flex;
+  gap: var(--pf-t--global--spacer--xs);
```
Comment (mcoker) [inline src/patternfly/components/ClipboardCopy/clipboard-copy.scss:66]: I think this should be a component var

```suggestion
  gap: var(--#{$clipboard-copy}__group--Gap);
```

with `--#{$clipboard-copy}__group--Gap: var(--pf-t--global--spacer--xs);` in the component vars

**Review by mcoker — COMMENTED** [review]
LGTM, just one nit!

File: src/patternfly/components/ClipboardCopy/clipboard-copy.scss [hunk 3]
```diff
@@ -1,40 +1,44 @@
...
+  --#{$clipboard-copy}__expandable-content--BackgroundColor: var(--pf-t--global--background--color--primary--default);
+  --#{$clipboard-copy}__expandable-content--BorderTopWidth: var(--pf-t--global--border--width--control--default);
+  --#{$clipboard-copy}__expandable-content--BorderRightWidth: var(--pf-t--global--border--width--control--default);
+  --#{$clipboard-copy}__expandable-content--BorderBottomWidth: var(--pf-t--global--border--width--control--default);
+  --#{$clipboard-copy}__expandable-content--BorderLeftWidth: var(--pf-t--global--border--width--control--default);
+  --#{$clipboard-copy}__expandable-content--BorderColor: var(--pf-t--global--border--color--default);
+  --#{$clipboard-copy}__expandable-content--BorderRadius: var(--pf-t--global--border--radius--small);
+  --#{$clipboard-copy}__expandable-content--OutlineOffset: var(--pf-t--global--spacer--xs);
+}
+.#{$clipboard-copy} {
```
Comment (thatblindgeye) [inline src/patternfly/components/ClipboardCopy/clipboard-copy.scss:24]: Starting about here it looks like some CSS vars need to eb moved to the block above

File: src/patternfly/components/ClipboardCopy/clipboard-copy.scss [hunk 4]
```diff
@@ -1,40 +1,43 @@
...
-  --#{$clipboard-copy}__actions-item--button--PaddingLeft: var(--#{$pf-global}--spacer--sm);
+  --#{$clipboard-copy}__actions-item--MarginTop: calc(-1 * var(--pf-t--global--spacer--xs));
+  --#{$clipboard-copy}__actions-item--MarginBottom: calc(-1 * var(--pf-t--global--spacer--xs));
+  --#{$clipboard-copy}__actions-item--button--PaddingTop: var(--pf-t--global--spacer--xs);
+  --#{$clipboard-copy}__actions-item--button--PaddingRight: var(--pf-t--global--spacer--xs);
+  --#{$clipboard-copy}__actions-item--button--PaddingBottom: var(--pf-t--global--spacer--xs);
+  --#{$clipboard-copy}__actions-item--button--PaddingLeft: var(--pf-t--global--spacer--xs);
 
   // Toggle button expanded
   &.pf-m-expanded {
```
Comment (thatblindgeye) [inline src/patternfly/components/ClipboardCopy/clipboard-copy.scss:43]: Real quick update, I may have misworded my previous comment but looks like we lost the closing curly brace for the CSS vars and the opening `.#{$clipboard-copy} {` for assigning the styles:

```suggestion
  --#{$clipboard-copy}__actions-item--button--PaddingLeft: var(--pf-t--global--spacer--xs);
}

.#{$clipboard-copy} {
  // Toggle button expanded
  &.pf-m-expanded {
```

**Review by mcoker — APPROVED** [review]

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**